roles/powervs_install_services/tasks/main.yml

```diff
@@ -96,17 +96,24 @@
         state: latest
 
     # modify /etc/chronyd file
-    - name: Replace iburst line in /etc/chronyd file
+    - name: Replace iburst line in /etc/chrony.conf file
       replace:
         path: /etc/chrony.conf
         regexp: "^(.*)local stratum 10$"
         replace: "local stratum 10"
 
-    - name: Edit allowed subnet in /etc/chronyd file
+    - name: Edit allowed subnets in /etc/chrony.conf file
```

Comment:
Add all private IPv4 addresses in allowed list in /etc/chrony.conf file

Reply:
Please fix this as well

Reply:
fixed